.prettierignore

```diff
@@ -0,0 +1 @@
+public/maps/
```

Comment:
@ssonal 

It is better to prettify the maps JSON also. There can be partial modifications to maps json in future, in that case diff does not show properly if not prettified. 

Below is the example of such commit done by one of the contributors,

https://github.com/covid19india/covid19india-react/pull/283/commits/1bb93ee02f52943ad5b88ffa2e8333a0b8be5a32

Reply:
Better way to speed up the pre-commit checks is to run the scripts only on the staged files instead of all files in the project using "lint-staged" configurations.

Reply:
Also it looks like I'm having trouble getting lint-staged to work? I've added the configuration but somehow when staging the files don't seem to lint, I'm not sure if I'm doing it wrong or if there's an issue. If you'd like to look into it, feel free to do so!

Reply:
@jeremyphilemon  Addressed it in the below PR
https://github.com/covid19india/covid19india-react/pull/364

Please review

Reply:
Dropped this commit from my PR. Best to handle it separately.